build.sbt

```diff
@@ -88,6 +88,10 @@ val libraryapi  = "io.github.scalan" %% "library-api" % specialVersion
 val libraryimpl = "io.github.scalan" %% "library-impl" % specialVersion
 val libraryconf = "io.github.scalan" %% "library-conf" % specialVersion
 
+val circeVersion = "0.11.1"
+val circecore = "io.circe" %% "circe-core" % circeVersion
+val circegeneric = "io.circe" %% "circe-generic" % circeVersion
+
```

Comment:
I suggest to not change circe version if possible. 
Instead if there are clear reasons, create an issue in ergo with some explained motivation.

Reply:
there wasn't `circe` dependency in `sigmastate`. i will change to ergo's version of `circe`